### DISCUSSION TITLE
Locked on pending transactions

### DISCUSSION THREAD

**Discussion by fcarpioc** [iotaledger/firefly#5177] [Help & Support]
I have tried to send my funds to the tanglePlay wallet and I have only managed to get two out of ten that I have made. Now I have all the funds blocked even though I indicated in the transaction that they have a maximum duration of 1 hour. In assets, all the SMRs appear and in balance it appears at 0, if I enter the balance configuration, it indicates that they are blocked in pending transactions. What can I do to unlock them? I did these operations days ago and it's still the same.

**Comment by CocodyRasta** [comment]
I have the same issue. Sorry I don't have an answer yet but I was hoping maybe you found a solution you could share?
